trucks.js

```diff
@@ -1,504 +1,610 @@
+      Facebook: 'https://www.facebook.com/pages/Ezells-Express/119007364918927',
+      Twitter: 'https://twitter.com/EzellsExpress1'
+  },
+  {
+      name: 'Falafel Salam',
+      type: ['Middle Eastern'],
+      schedule: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Sunday'],
+      description: 'Gourmet, authentic falafel & shawarma',
+      website: 'http://falafelsalam.com',
+      Facebook: 'https://www.facebook.com/FalafelSalam',
+      Twitter: 'https://twitter.com/FalafelSalam'
+  },
+  {
+      name: 'Fez',
+      type: ['Mediterranean'],
+      schedule: ['Tuesday', 'Wednesday', 'Thursday', 'Friday'],
+      payment: ['Cash', 'Cards'],
+      description: 'Modern Middle eastern street food (and drink)!',
+      Facebook: 'https://www.facebook.com/FezOnWheels/info',
+      Twitter: 'https://twitter.com/fezonwheels'
+  },
+  {
+      name: 'Flair Taco',
+      type: ['Mexican'],
+      schedule: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'],
+      payment: ['Cash'],
+      description: 'Regular taco truck in the Fremont neighborhood - open 7 days a week and late night',
+      Facebook: 'https://www.facebook.com/pages/Flair-Taco/197832600252403'
+  },
+  {
+      name: 'Frank\'s Franks',
+      type: ['Hot Dogs'],
+      schedule: ['Monday', 'Sunday'],
+      payment: ['Cash', 'Cards'],
+      description: 'Gourmet hot dogs, weiners, sausages, hot links and more, contact at 425.761.9447 or franksfranks3@gmail.com',
+      Facebook: 'https://www.facebook.com/franksfrankshotdogs'
+  },
+  {
+      name: 'Fruit Chatter Box',
+      type: ['Sweets'],
+      schedule: ['Sunday'],
+      payment: ['Cash', 'Cards'],
+      description: 'Fruit Chatter Box provides fresh, seasonal fresh fruit in cups or on a stick at local festivals and street fairs.',
+      Facebook: 'https://www.facebook.com/fruitchatterbox',
+      Twitter: 'https://twitter.com/fruitchatterbox'
+  },
+  {
+      name: 'Full Tilt Ice Cream',
+      type: ['Sweets'],
+      payment: ['Cash', 'Cards'],
+      description: 'Handmade ice creams!',
+      website: 'http://www.fulltilticecream.com',
+      Facebook: 'https://www.facebook.com/fulltilticecreams',
+      Twitter: 'https://twitter.com/fticecream'
+  }
+];
+
+
+// this module should support the following methods:
+// getTrucks() - return all trucks
+// getTruck(name) - return the truck object matching 'name'
+// getFoodTypes() - return unique list of all associated food types (underscore has a function to help)
+// filterByDay(day) - return trucks with 'day' in schedule (use your filterByDay function from Module 3 homework)
+// filterByFoodType(foodType) - return trucks with associated 'foodType'
+
+//Extend the trucks module, trucks.js provided with the following methods:
+//getTrucks This method takes no parameters and returns a list of all the truck objects in the module.
+
+function getTrucks() {
+    return foodTrucks;
+}
+
+var getTrucks = function () {

```

Comment:
This function should just return the entire trucks list. By only returning the names, it breaks the client-side code. If you run the server, and load the page in a browser, you'll see the list of trucks is just a list of undefined. That's because the truck objects with name properties were never returned.